npm/vue/src/index.ts

```diff
@@ -53,17 +58,162 @@ Cypress.on('run:start', () => {
     const document: Document = cy.state('document')
     let el = document.getElementById(ROOT_ID)
 
+    if (!el) {
+      throw Error(`no element found at query #${ROOT_ID}. Please use the mount utils to mount it properly`)
+    }
+
     el.innerHTML = ''
     document.head.innerHTML = initialInnerHtml
   })
 })
 
-export function mount<Props = any> (
-  comp: Component<Props>,
-  options: CyMountOptions<Props> = {},
+type PublicProps = VNodeProps & AllowedComponentProps & ComponentCustomProps;
+
+// Class component - no props
+export function mount<V>(
```

Comment:
Can we put a link to where these came from? So if someone looks at in the future and wants to update they can copy paste again - actually reading this type is basically impossible unless you know Vue 3 inside-out

Reply:
Done !!